Makefile

```diff
@@ -297,22 +299,22 @@ push: k8s-broker-push user-broker-push controller-manager-push apiserver-push
 
 k8s-broker-push: k8s-broker-image
 	[ ! -z "$(REGISTRY)" ] || (echo Set your REGISTRY env var first ; exit 1)
-	docker tag k8s-broker:$(VERSION) $(REGISTRY)/k8s-broker:$(VERSION)
-	docker push $(REGISTRY)/k8s-broker:$(VERSION)
+	docker tag $(IMAGES_PREFIX)k8s-broker:$(VERSION) $(REGISTRY)/$(IMAGES_PREFIX)k8s-broker:$(VERSION)
+	docker push $(REGISTRY)/$(IMAGES_PREFIX)k8s-broker:$(VERSION)
 
 user-broker-push: user-broker-image
 	[ ! -z "$(REGISTRY)" ] || (echo Set your REGISTRY env var first ; exit 1)
-	docker tag user-broker:$(VERSION) $(REGISTRY)/user-broker:$(VERSION)
-	docker push $(REGISTRY)/user-broker:$(VERSION)
+	docker tag $(IMAGES_PREFIX)user-broker:$(VERSION) $(REGISTRY)/$(IMAGES_PREFIX)user-broker:$(VERSION)
+	docker push $(REGISTRY)/$(IMAGES_PREFIX)user-broker:$(VERSION)
 
 controller-manager-push: controller-manager-image
 	[ ! -z "$(REGISTRY)" ] || (echo Set your REGISTRY env var first ; exit 1)
-	docker tag controller-manager:$(VERSION) $(REGISTRY)/controller-manager:$(VERSION)
-	docker push $(REGISTRY)/controller-manager:$(VERSION)
+	docker tag $(IMAGES_PREFIX)controller-manager:$(VERSION) $(REGISTRY)/$(IMAGES_PREFIX)controller-manager:$(VERSION)
+	docker push $(REGISTRY)/$(IMAGES_PREFIX)controller-manager:$(VERSION)
 
 apiserver-push: apiserver-image
 	[ ! -z "$(REGISTRY)" ] || (echo Set your REGISTRY env var first ; exit 1)
-	docker tag apiserver:$(VERSION) $(REGISTRY)/apiserver:$(VERSION)
-	docker push $(REGISTRY)/apiserver:$(VERSION)
-	docker tag apiserver:$(VERSION) $(REGISTRY)/apiserver:latest
-	docker push $(REGISTRY)/apiserver:latest
+	docker tag $(IMAGES_PREFIX)apiserver:$(VERSION) $(REGISTRY)/$(IMAGES_PREFIX)apiserver:$(VERSION)
+	docker push $(REGISTRY)/$(IMAGES_PREFIX)apiserver:$(VERSION)
+	docker tag $(IMAGES_PREFIX)apiserver:$(VERSION) $(REGISTRY)/$(IMAGES_PREFIX)apiserver:latest
+	docker push $(REGISTRY)/$(IMAGES_PREFIX)apiserver:latest
```

Comment:
Let's not push to `latest`. [Here](https://github.com/kubernetes-incubator/service-catalog/issues/504#issuecomment-286204441) is what I think we should do instead.